src/pymortests/strategies.py

```diff
@@ -473,3 +475,17 @@ def _eigs():
 def equal_tuples(draw, strategy, count):
     val = draw(strategy)
     return draw(hyst.tuples(*[hyst.just(val) for _ in range(count)]))
+
+
+# stick to a few representative examples to avoid only seeing degenerate cases
+# in the selected examples
+mus = hyst.dictionaries(
+    keys=hyst.sampled_from(['t', 'foo', 'bar']),
+    values=hyst.sampled_from([
+        np.array([1.]),
+        np.array([1., 32., 3]),
+        ExpressionFunction('x+1', 1, (1,)),
+        ExpressionFunction('[1., 0] * x + [0, 1.] * x**2', 1, (2,)),
```

Comment:
It seems that, to define a time-dependent function, a variable `x` needs to be used, which feels a bit strange.

Reply:
Yes I agree. But I do not think it is warranted to add additional complexity to `ExpressionFunction` just to support a different name for the variable.

Reply:
Ok. Could this be documented somewhere additionally?

Reply:
I have just added an remark to `Parameters.parse`.